common/service/config/config.go

```diff
@@ -237,13 +242,39 @@ type (
 	}
 
 	// Address indicate the remote cluster's service name and address
+	//
+	// Deprecated: please use ClusterInformation instead
 	Address struct {
 		// RPCName indicate the remote service name
 		RPCName string `yaml:"rpcName"`
 		// Address indicate the remote service IP address
 		RPCAddress string `yaml:"rpcAddress"`
 	}
 
+	// ClustersInformation contains the all cluster which participated in cross DC
+	ClustersInformation struct {
```

Comment:
Can we use any other name other than ClustersInfomation. The reason is we have ClusterInfomation. For example: GlobalReplicationConfig